src-ui/js/ui/ToolArea.js

```diff
@@ -352,6 +352,9 @@ ui.toolarea = {
 	flushexcell: function() {
 		ui.puzzle.board.flushexcell();
 	},
+	traceroute: function() {
+		ui.puzzle.board.operate("traceroute");
```

Comment:
Making "Trace Route" an operation seems like a bad idea... I was expecting it to just be an Input mode like Roma's "Check Route" option. This only shows one route at a time (which is much clearer), and is a much less disruptive change to the system as a whole.

Reply:
Ohh, that's exactly what I wanted but didn't know of any example! Thanks for the suggestion. Rampage now uses the same "check route" option as Roma.